plugins/client/graphql-kotlin-client-generator/src/main/kotlin/com/expediagroup/graphql/plugin/client/generator/GraphQLClientGenerator.kt

```diff
@@ -228,6 +229,13 @@ class GraphQLClientGenerator(
     }
 }
 
+/**
+ * This is the reccommended approach now with the deprecation of String.capitalize from the
+ * Kotlin stdlib in version 1.5.
+ */
+internal fun String.capitalizeWithDefaultLocale(): String =
+    replaceFirstChar { if (it.isLowerCase()) it.titlecase(Locale.getDefault()) else it.toString() }
```

Comment:
should this be also using `uppercase` instead of `titlecase` (for consistency)

Reply:
Yep, this was autogenerated from deprecation warning